### PR TITLE
feat: Add option to count all notifications

### PR DESCRIPTION
**Pull request by bjagg**
Resovles #235

### CODE REVIEW

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 1]
```diff
@@ -105,13 +109,17 @@ export default {
...
+        return this.notifications.length;
+      } else {
+        return this.notifications.filter(
+          ({ attributes }) => !JSON.parse(attributes?.READ?.[0] || "true")
+        ).length;
+      }
     },
     variant() {
-      return this.unreadCount < 1 ? "default" : "danger";
+      return this.count < 2 ? "default" : "danger";
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]: :question: Why is having a single notification ignored from the stylistic highlight?

Comment (bjagg) [reply notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]: Great catch! That was cut&paste regression

File: notification-portlet-webcomponents/notification-icon/package-lock.json [hunk 2]
```diff
@@ -1760,7 +1760,7 @@
     },
     "acorn-jsx": {
       "version": "3.0.1",
-      "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
+      "resolved": "http://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/package-lock.json]: Making registry links http only? :confused:

Comment (bjagg) [reply notification-portlet-webcomponents/notification-icon/package-lock.json]: Hmm, I don't edit those files. Wonder how it changed to `http`?

Comment (jonathanmtran) [reply notification-portlet-webcomponents/notification-icon/package-lock.json]: It probably changed when `npm install` was executed. I haven't dug into why this is, but I've seen this happen ...

Comment (bjagg) [reply notification-portlet-webcomponents/notification-icon/package-lock.json]: I'll revert these to `https` soon

**Comment by bjagg** [comment]
Just completed testing all these little changes. Still works!
Merging....

**Pull request merged**